cmd/ingester/app/processor/span_processor.go

```diff
@@ -18,6 +18,7 @@ import (
 	"context"
 	"fmt"
 	"io"
+	"log"
```

Comment:
we don't use log package, only zap

Reply:
I see

Tell me about it.
Because span_processor does not hold the logger object of zap,
We need to propagate zap objects from parallel_processor to span_processor.
The interface changes, is this acceptable?

Or should I create a new logger in span_processor as follows?
https://github.com/jaegertracing/jaeger/blob/main/cmd/flags/service.go#L93-L96

Reply:
`*zap.Logger` is only created in `main()` and passed through to all components that need it. It's ok to change the component constructor's interface to accept a logger.